packages/kbn-test/src/functional_tests/cli/run_tests_cli.js

```diff
@@ -0,0 +1,63 @@
+import getopts from 'getopts';
+import { runTests } from '../../';
+import {
+  FUNCTIONAL_CONFIG_PATH,
+  API_CONFIG_PATH,
+} from '../../functional_tests/lib';
+
+const defaultConfigPaths = [FUNCTIONAL_CONFIG_PATH, API_CONFIG_PATH];
+
+/**
+ * Run servers and tests for each config
+ * Only cares about --config option. Other options
+ * are passed directly to functional_test_runner, such as
+ * --bail, --verbose, etc.
+ * In the future, this method will accept --tests-only,
+ * --servers-only.
+ * @param {configPaths}     array of paths to configs
+ */
+export function runTestsCli() {
+  const {
+    configs,
+    help,
+  } = processArgv();
+
+  if (help) return displayHelp();
+
+  runTests(configs);
+}
+
+
+function processArgv() {
+  const options = getopts(process.argv.slice(2)) || {};
+  const configs = [];
+
+  try {
+    let configOptions = options.config;
+    configOptions = typeof configOptions === 'string'
+      ? [configOptions]
+      : configOptions;
+    configs.push(...configOptions);
+  } catch (err) {
```

Comment:
I'm concerned that this is ignoring the error here. I would normally do something like this:

```js
const configs = options.config
  ? [].concat(options.config || []) // will add options.config to the array if it is not an array, or add all of its items if it is
  : defaultConfigPaths
```

Something like this uses less mutation, doesn't hide any errors, and I think is a little more obvious.